tests/show_techsupport/test_techsupport_no_secret.py

```diff
@@ -92,8 +92,8 @@ def test_secret_removed_from_show_techsupport(
     list_command = "ls {0}/etc/pam_radius_auth.d/*.conf || true".format(dump_extract_path)
     config_file_list = duthost.shell(list_command)["stdout_lines"]
     for config_file in config_file_list:
-        sed_command = "sed -nE '/{0}/P' {1}/etc/pam_radius_auth.d/{2}"\
-            .format(radius_passkey, dump_extract_path, config_file)
+        sed_command = "sed -nE '/{0}/P' {1}"\
+            .format(radius_passkey, config_file)
```

Comment:
can we manually create a file in /etc/pam_radius_auth.d/?
Do you think aaa test should clean up all the configuration?

Reply:
We can't manually create this file, it's create by hostcfgd.

The issue is not caused by ConfigUpdater AAA test. It's a bug in hostcfgd: when radius config changed, the old config file doesn't delete.